applications/asset_tracker/README.rst

```diff
@@ -196,6 +196,10 @@ Alternatively, use the command line tool ``menuconfig`` or configure the options
 
 .. external_antenna_note_end
 
+.. note::
+   When ``CONFIG_BOOTLOADER_MCUBOOT=y`` this application is adds an overlay configuration for MCUboot which enables RSA signing of the images for backwards compatbility with MCUboot versions pre NCS 1.4.
```

Comment:
Redundant `is`: `...application is adds...`

Reply:
fixed